### PR TITLE
Fix the taiko scroller bar appearing during cinema mod

### LINKED ISSUE #15869: taiko-slider should be hidden when using cinema mod
### Discussed in https://github.com/ppy/osu/discussions/15866

<div type='discussions-op-text'>

<sup>Originally posted by **Joppe27** November 30, 2021</sup>
The goal of the cinema mod is being able to watch the video that comes with a beatmap. The taiko-slider and taiko-slider-fail elements should probably be hidden (stable behavior) as they cover up the video which defeats the point of the cinema mod: 
osu_2021-11-29_22-07-56
This was probably overlooked/forgotten because the osu!lazer default skin doesn't use this element anymore.</div>

Prio-3 because I don't think cinema mod is used enough that this needs a timely fix.

### PR DESCRIPTION
**Pull request by MyDogsEd**
The taiko-slider is not included in  `Playfield`, so it doesn't get hidden when calling `drawableRuleSet.Playfield.Hide()`.   Calling `drawableRuleSet.Hide()` hides the taiko-slider, in addition to the rest of the `Playfield`.

Resolves #15869

### CODE REVIEW

**Comment by peppy** [comment]
This.. surprisingly works very well. I've tested in each ruleset and it doesn't seem to have any adverse effects. `DrawableRuleset` does contain a few overlay elements, but all are related to gameplay, and in the case of `cinema` mod we don't care about these.

Because this is manual testing I'll wait for a second opinion before merging to make sure I haven't missed anything.

**Review by peppy — APPROVED** [review]

**Review by Joehuu — APPROVED** [review]
LGTM

**Pull request merged**
